src/IssueViz.Security/Hotspots/HotspotsStore.cs

```diff
@@ -65,14 +59,22 @@ public IAnalysisIssueVisualization GetOrAdd(IAnalysisIssueVisualization hotspotV
                 return existingHotspot;
             }
 
+            var oldIssues = hotspots.ToArray();
+
             hotspots.Add(hotspotViz);
 
+            NotifyIssuesChanged(oldIssues);
+
             return hotspotViz;
         }
 
         public void Remove(IAnalysisIssueVisualization hotspotViz)
         {
+            var oldIssues = hotspots.ToArray();
+
             hotspots.Remove(hotspotViz);
+
+            NotifyIssuesChanged(oldIssues);
```

Comment:
This seems odd too - why are we saying "everything's changed" when only one issue has been removed?